templates/pages/index.html

```diff
@@ -5,9 +5,13 @@
 {% block content %}
 	<!-- Main jumbotron for a primary message -->
 	<div class="jumbotron">
-		<h1>TrackCat</h1>
+        <div class="h1"> 
```

Comment:
There's no need to have an extra `<div>` here - in CSS, you can define the behavior of built-in HTML tags like `<h1>` by calling them without the dot in front. For example, for a really big h1:

    h1 {
          font-size:100px;
    }